tests/test_footnotes.py

```diff
@@ -22,8 +22,13 @@ def test_annotate_footnotes_works(m_16_19_doc):
     assert notes[:1] == [
         (1, 'The FDCCI was first established by OMB “Memo for CIOs: '
             'Federal Data Center Consolidation Initiative,” issued '
-            'on February 26, 2010, and modified by subsequent memoranda. '),
+            'on February 26, 2010, and modified by subsequent memoranda.'),
     ]
+    # This is to test that we don't annotate small text as footnotes unless
+    # that text is in the bottom third of the page--we know that "Total Server
+    # Count" is small text in the middle of a page in M-16-19 and that it
+    # would show up as part of footnote 26 if we didn't guard against that:
+    assert "Total Server Count" not in notes[25][1]
```

Comment:
This seems okay for now, though I know I've personally seen pages outside of the OMB policy context that had footnotes extending above the bottom third of the page, so I'm not sure if this will hold for all policies or not... guess we'll see!

Another option in the future might be to detect the horizontal line that seems to demarcate the footnotes in most of the memos, but we can deal with that if/when we need to...